### PR TITLE
chore: use prettier for code formatting

### PR DESCRIPTION
**Pull request by henri-hulski**

### CODE REVIEW

**Comment by gaspard** [comment]
This is really cool @henri-hulski ! Everything will be a tad different but so much more consistent...

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  ],
+  "rules": {
+    "prettier/prettier": [
+      "error",
+      {
+        "trailingComma": "es5",
+        "singleQuote": true,
+        "bracketSpacing": false,
+        "semi": false,
+        "bracketSpacing": true
```
Comment (gaspard) [inline .eslintrc.json]: bracketSpacing twice in settings...

File: package.json [hunk 2]
```diff
@@ -4,22 +4,22 @@
...
     "setup": "node ./scripts/run.js prepublish nodemos",
-    "test": "node ./scripts/run.js test"
+    "build:demos": "node ./scripts/run.js prepublish demos",
+    "link": "node ./scripts/run.js link",
+    "postsetup": "npm run link",
+    "checkdeps": "node ./scripts/dependencies.test.js",
+    "pretest": "npm run checkdeps",
+    "test": "node ./scripts/run.js test",
+    "deploy": "cd website/publisher && npm run ci",
+    "format": "prettier-eslint --write \"**/*.js\"&&prettier-eslint --write \"packages/node_modules/**/*.js\""
```
Comment (gaspard) [inline package.json]: Please keep scripts sorted...

File: package.json [hunk 3]
```diff
@@ -56,23 +62,9 @@
...
-    "ignore": [
-      "/node_modules/**",
-      "/packages/*/**/node_modules/**",
-      "**/build/",
-      "**/*.min.js",
-      "**/bundle.js",
-      "coverage/**",
-      "vendor/**"
-    ]
-  },
```
Comment (gaspard) [inline package.json]: Yeheee ! Won't regret this one.

File: package.json [hunk 4]
```diff
@@ -94,17 +95,19 @@
...
     "node-sass": "4.5.3",
     "npm-run-all": "^4.0.2",
     "nyc": "^10.3.2",
+    "prettier": "^1.3.1",
+    "prettier-eslint-cli": "^4.0.2",
     "prop-types": "15.5.10",
     "react-addons-test-utils": "^15.4.2",
     "react-scripts": "^1.0.6",
     "shx": "^0.2.2",
-    "standard": "github:gaspard/standard#v8.6.0fix2",
```
Comment (gaspard) [inline package.json]: Hahah, this fix didn't live long. Cool.

File: packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js [hunk 5]
```diff
@@ -1,15 +1,15 @@
 /* eslint-disable */
-import {connect as Cerebral} from 'cerebral/react';
-import {limitedList} from "./limitedList";
-import {anotherComputed} from "./anotherComputed";
+import {connect as Cerebral} from "cerebral/react"
```
Comment (gaspard) [inline packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js]: Why isn't single quote rule respected here ?

File: package.json (hunk 2 above)
Comment (henri-hulski) [inline package.json]: @gaspard I tried to sort them logically. ;-)

File: packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js (hunk 5 above)
Comment (henri-hulski) [inline packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js]: `packages/codemods/1.x-2.x/__testfixtures__` is excluded from linting as it uses decorators whhich are not compatible with prettier.

File: .eslintrc.json (hunk 1 above)
Comment (henri-hulski) [inline .eslintrc.json]: Removed it entirely as it's the default.

File: packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js (hunk 5 above)
Comment (henri-hulski) [inline packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js]: `single quote rule`? Or do you mean `bracketSpacing`?
I ran prettier once before excluding it but changed the `bracketSpacing` rule after.
I can fix that.

Comment (edgesoft) [inline packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js]: "singleQuote": true

Comment (henri-hulski) [inline packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js]: Does not work. Seems they are also excluded on file base.
Will leave them as they're not part of code base.

Comment (henri-hulski) [inline packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js]: Ah now I see. Yeah this directory is not linted at all.

Comment (henri-hulski) [inline packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js]: If the fixtures have to look exactly like the resulted code we should fix this by hand.
Maybe after this PR is merged.

File: package.json (hunk 2 above)
Comment (Guria) [inline package.json]: @henri-hulski it's very hard to reason logic in this case. I have tried it and there always was ambiguous result. So seems alphabetic order is only option to have stable unopinionated order.

File: packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js (hunk 5 above)
Comment (gaspard) [inline packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js]: OK, no problem.

**Comment by edgesoft** [comment]
Awesome!

**Pull request merged**